server/go/api_apis.go

```diff
@@ -19,7 +19,7 @@ import (
 
 // ApisApiController binds http requests to an api service and writes the service results to the http response
 type ApisApiController struct {
-	service      ApisApiServicer
```

Comment:
go fmt ./... will have this aligned correctly no need for so many changes

Reply:
Resolved. Did `make format`.

Please mark as resolved, if you are happy with it.